pysteps/io/exporters.py

```diff
@@ -510,14 +510,14 @@ def initialize_forecast_exporter_netcdf(
     pr = pyproj.Proj(metadata["projection"])
     lon, lat = pr(x_2d.flatten(), y_2d.flatten(), inverse=True)
 
-    var_lon = ncf.createVariable("lon", np.float, dimensions=("y", "x"))
+    var_lon = ncf.createVariable("lon", float, dimensions=("y", "x"))
     var_lon[:] = lon.reshape(shape)
     var_lon.standard_name = "longitude"
     var_lon.long_name = "longitude coordinate"
     # TODO(exporters): Don't hard-code the unit.
     var_lon.units = "degrees_east"
 
-    var_lat = ncf.createVariable("lat", np.float, dimensions=("y", "x"))
+    var_lat = ncf.createVariable("lat", float, dimensions=("y", "x"))
```

Comment:
Numpy introduced the following deprecation warning:
```
DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
```

So the change was simply meant to silence the warning without changing the behavior of the method. It is true that float32 might be a better solution for lat lon coordinates (@pulkkins ?), but I would suggest to not include such change in this PR.